include/boost/gil/image.hpp

```diff
@@ -101,6 +101,19 @@ class image {
        allocate_and_copy(img.dimensions(),img._view);
     }
 
+    // TODO Optimization: use noexcept (requires _view to be nothrow copy constructible)
+    image(image&& img) :
+      _view(img._view),
+      _memory(img._memory),
+      _align_in_bytes(img._align_in_bytes),
+      _alloc(std::move(img._alloc)),
```

Comment:
Quoting [Arthur O'Dwyer](https://www.packtpub.com/gb/application-development/mastering-c17-stl) here, since 

> [allocator is just a thin "handle"](https://www.youtube.com/watch?v=0MdSJsCTRkY) (...) Copying an allocator type should always be relatively cheap. Still, using std::move here wouldn't have hurt.

So, let's assume it doesn't.